docs/docs/how-to/functions/getting-started.md

```diff
@@ -83,6 +76,20 @@ export default function handler(req, res) {
 }
 ```
 
+## Headers
```

Comment:
The example seems off?

Reply:
😂 That was by accident. Thank you for catching it! Cheers